packages/repository/src/model.ts

```diff
@@ -47,6 +47,20 @@ export interface PropertyDefinition {
   [attribute: string]: any; // Other attributes
 }
 
+/**
+ * Defining the settings for a model
+ */
+export interface ModelSettings {
+  description?: string;
+  forceId?: boolean;
+  hiddenProperties?: string[];
+  scope?: object;
+  strict?: boolean;
```

Comment:
Have you considered adding tsdoc comments for individual settings? A link to loopback.io docs where we are describing model settings can be a good start.

Reply:
The link: https://loopback.io/doc/en/lb4/Model.html#supported-entries-of-model-definition

Reply:
Done